src/index.css

```diff
@@ -1,13 +1,17 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+
+@font-face{
+  font-family: "Nexa";
+  src: url("../src/assets/fonts/NexaLight.otf") format("opentype");
+  font-weight: normal;
```

Comment:
I would like you to research how fonts can be changed using theming: https://v3.material-ui.com/customization/themes/#typography

Reply:
Noted, we added chore in PT for that.